vulcanai/datasets/tabulardataset.py

```diff
@@ -345,14 +345,15 @@ def reverse_create_one_hot_encoding(self, column_name=None,
         logger.info("Successfully converted %d \
                     columns back from dummy format.", len(dummy_tuples))
 
-    def identify_sufficient_non_null(self, threshold):
+    def identify_null(self, threshold):
```

Comment:
this restructuring and rethinking of this function makes a lot more sense. Great work!